CHANGELOG.md

```
@@ -1,45 +1,29 @@
 # [1.5.0](https://github.com/VilnaCRM-Org/website/compare/v1.4.0...v1.5.0) (2025-05-11)
 
-
 ### Features
 
-* **#178:** dependabot workflow  ([#179](https://github.com/VilnaCRM-Org/website/issues/179)) ([bc20712](https://github.com/VilnaCRM-Org/website/commit/bc207126629fda72074a7b5d49d80762bfa0b3b6)), closes [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178)
-
-
+- **#178:** dependabot workflow ([#179](https://github.com/VilnaCRM-Org/website/issues/179)) ([bc20712](https://github.com/VilnaCRM-Org/website/commit/bc207126629fda72074a7b5d49d80762bfa0b3b6)), closes [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 100; Actual: 496

(MD013, line-length)